### PR TITLE
GUVNOR-2990: Reproduce issue with invalid missing range messages

### PR DESCRIPTION
**Pull request by jomarko**

### CODE REVIEW

File: drools-wb-services/drools-wb-verifier/drools-wb-verifier-backend/src/test/java/org/drools/workbench/services/verifier/webworker/client/DecisionTableAnalyzerMultipleIssuesFromFileTest.java [hunk 1]
```diff
@@ -54,6 +56,14 @@ public void setUp() throws
     }
 
     @Test
+    public void testGapAnalysis() throws Exception {
```
Comment (karreiro) [inline drools-wb-services/drools-wb-verifier/drools-wb-verifier-backend/src/test/java/org/drools/workbench/services/verifier/webworker/client/DecisionTableAnalyzerMultipleIssuesFromFileTest.java]: Hi @jomarko 

I think that you need to add the `@Ignore` here. ;-)

Comment (jomarko) [inline drools-wb-services/drools-wb-verifier/drools-wb-verifier-backend/src/test/java/org/drools/workbench/services/verifier/webworker/client/DecisionTableAnalyzerMultipleIssuesFromFileTest.java]: @karreiro you are right. Thanks. Done.

Comment (karreiro) [inline drools-wb-services/drools-wb-verifier/drools-wb-verifier-backend/src/test/java/org/drools/workbench/services/verifier/webworker/client/DecisionTableAnalyzerMultipleIssuesFromFileTest.java]: Thanks!

**Comment by mariofusco** [comment]
Test case included in and fixed by this https://github.com/kiegroup/drools-wb/pull/451

**Pull request closed without merging**